new_core_concepts/modules/ROOT/pages/typeql/query-clauses.adoc

```diff
@@ -61,10 +61,18 @@ match $p isa person, has name $n; $n == "John";
 delete has $n of $p; # Delete the person instance's ownership of the name "John"
 ----
 
-Any concepts which are deleted will be removed from the stream.
+Any deleted concept is removed from the stream.
 
 === update
-Used to update `@card(0..1)` or `@card(1)` ownerships.
+Update clauses are used as an alternative to `delete` + `insert` when you need to ensure
+that the owned attribute (`has`) or the linked role player (`links`) is the correct
+concept, regardless of the previous state.
+
+The returned result contains existing incoming data and updated concepts (similarly to the result of
+a potential `delete` + `insert` pipeline).
+
+As `update` can insert only a single connection for a type, for your safety, it is only allowed when the interface
+is defined with cardinality limited to 1: `@card(0..1)` (default for `owns` and `relates`) or `@card(1)`.
```

Comment:
Too much detail. Look at what poor `put` gets.

Reply:
I don't see why the details are needed here? This page does not need to spell out the behaviour of an update apart from how it affects the pipeline. 
The fact that it's the cardinality for owns or relates is irrelevant. "For your safety" is not explained, so why mention it? 
> As `update` can insert only a single connection for a type
Why?

> The returned result contains existing incoming data and updated concepts (similarly to the result of
a potential `delete` + `insert` pipeline).
Wouldn't the stream actually remain untouched?